meshnets/process.py

```diff
@@ -47,13 +50,20 @@ def main(_):
 
     # Convert the .vtk files to graph data.
     for folder in sim_folders:
-        vtk_file_path = os.path.join(folder, 'pressure_field.vtk')
-        graph_data = data_processing.mesh_file_to_graph_data(vtk_file_path,
-                                                             WIND_VECTOR,
-                                                             load_pressure=True)
+        openfoam_mesh_path = os.path.join(folder, 'pressure_field.vtk')
```

Comment:
Yes, I can remove the hardcoded names. I had just left them like this because through out the project it is assumed that the data has the following structure:

```
    data_dir
        - 199834348
            - pressure_field.vtk
            - pressure_field.pt
        - 129383949
            - pressure_field.vtk
            - pressure_field.pt
```

See, for example, the previously implemented [dataset class](https://github.com/inductiva/meshnets/blob/f0597da2592c1321d4e264a82c7ade710b534bba/meshnets/utils/datasets.py#L14)